synapse/state/__init__.py

```diff
@@ -190,6 +190,7 @@ async def compute_state_after_events(
         room_id: str,
         event_ids: Collection[str],
         state_filter: Optional[StateFilter] = None,
+        await_full_state: bool = True,
```

Comment:
Let's update the docstring below too. Something like:
```
            await_full_state: if `True`, will block if we do not yet have complete state
                at the given `event_id`s, regardless of whether `state_filter` is
                satisfied by partial state.
```
We want to call out that `state_filter` is ignored, because in all the other functions that have an `await_full_state` parameter, `state_filter`s that are restrictive enough result in no blocking.